docs/schema/guidance/repr-beneficial-ownership.rst

```diff
@@ -0,0 +1,75 @@
+
+In BODS, the following properties are used to represent such information disclosure:
+
+* ``beneficialOwnershipOrControl`` (See :ref:`schema-interest`)
+* ``directOrIndirect`` (See :ref:`schema-interest`)
+* ``componentStatementIDs`` (See :ref:`schema-ownership-or-control-statement`)
+* ``isComponent`` (See :ref:`schema-entity-statement`)
+
+Requirements
+------------------------
+
+Representing beneficial ownership
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If a person is a beneficial owner of an entity, E, whether directly or indirectly, and one of them is required to declare this beneficial ownership:
+
+1. There MUST be a *primary* Ownership-or-control Statement connecting the two which represents the beneficial ownership relationship. In particular: 
+
+   a. the entity’s Statement MUST be the ``subject``;
+   b. the person’s Statement MUST be the ``interestedParty``;
+   c. ``isComponent`` MUST be false;
+   d. the ``interests`` which make the person meet the criteria for their being declared a beneficial owner MUST be included in this primary Ownership-or-Control Statement if known; and
+   e. the ``interests`` in (d) MUST have ``beneficialOwnershipOrControl`` set to 'true'.
+
+2. If beneficial ownership is known to be exercised indirectly, via intermediary entities or arrangements then the ``directOrIndirect`` MUST be ‘indirect’. If it is known to be exercised directly then ``directOrIndirect`` MUST be ‘direct’. Otherwise ``directOrIndirect`` MUST be ‘unknown’.
+
+Representing intermediaries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Where beneficial ownership is known to be exercised indirectly, via known intermediary entities or arrangements, this SHOULD be represented in addition to the above. In particular:
+
+1. The chain of known intermediary entities and arrangements SHOULD be represented by *secondary* Entity Statements, Ownership-or-control Statements and Person Statements.
+2. These secondary statements SHOULD link the beneficial owner’s Statement to entity E’s statement indirectly.
+3. These secondary statements SHOULD all have ``isComponent`` set to 'true'.
+4. These secondary statements SHOULD all have their ``statementID`` values listed in the ``componentStatementIDs`` array of the primary Ownership-or-control Statement.
+5. When the primary Ownership-or-control Statement is published in a BODS file:
+
+   a. all secondary statements referenced from ``componentStatementIDs`` MUST also be published in that file;
+   b. all secondary statements must appear before the primary Ownership-or-control Statement in the list of statements.
+
+Example
+--------
+
+In the following example, Person 1 is a beneficial owner of Company E. They exercise that beneficial ownership via an intermediary company, Company A. (Note: abbreviated statement IDs are used for brevity and clarity of explanation. Short IDs like ‘ooc-2’ are not valid in BODS.)
+
+.. figure:: ../../_assets/RepresentingChainsBODS-Statements.svg
```

Comment:
The text in this diagram is quite small and I found it difficult to read

Reply:
I've had a play with the image settings. I've now used a percentage width. That will mean that on some setups it will be too wide. But better that than too small. (I don't think there's any rst option to set a max-width.)